cumulusci/tasks/metadata_etl/base.py

```diff
@@ -305,3 +306,48 @@ def _transform(self):
                 removed_api_names.add(api_name)
 
         self.api_names = self.api_names - removed_api_names
+
+
+class UpdateMetadataFirstChildTextTask(MetadataSingleEntityTransformTask):
+    """
+    Updates the metadata's first child tag's text.
+    Does not support tags with complex/object-like values.
+    """
+
```

Comment:
When adding a task, part of the DoD is making sure that it is documented.

1. Add a task_docs attribute with a string (reStructuredText format) explaining what the task is used for and at least one example.
2. Run `make docs` to update docs/tasks.rst, build the Sphinx docs, and open them in your browser.
3. Check that everything is rendering as expected.

Reply:
Done.   This is super cool!